apps/sms/index.html

```diff
@@ -63,7 +63,7 @@
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia_sim_picker/style.css">
     -->
     <!-- Shared code -->
-    <script defer src="shared/js/l10n.js"></script>
+    <script type="application/javascript;version=1.8" defer src="shared/js/l20n.js"></script>
```

Comment:
yeah, unfortunately it is. Bug 932513. Hopefully to be fixed quite soon (SM team just landed one of the major dependencies)